### ISSUE TITLE
Thermal Expansion's adding pulverizer recipe crash on world load.

### ISSUE THREAD

**Issue by ivardensphere** [TeamMetallurgy/Metallurgy4#65]
This is in reference to 
https://github.com/TeamMetallurgy/Metallurgy4/issues/58
and
https://github.com/CoFH/Feedback/issues/158#issuecomment-75463001

I was having this exact crash with both metallurgy and Thermal Expansion. I decided to tear apart the pack I created to figure out what was causing it since no one's mod seems to be the culprit. 

I found out that if I put in bdew's advanced generators mod, it causes this crash every time.  
I am running a pack with 200+ mods and that one mod by bdew is the only one that causes this crash.

I found it kind of curious as to why that mod was cause the issue for metallurgy and thermal expansion.

**Comment by AegisLesha** [comment]
Interestingly enough, I didn't put any bdew's mods in my pack

**Comment by Glassmaker** [comment]
It's still unknown what causes this, Metallurgy doesn't register it's stuff with Thermal Expansion,  as Thermal expansion uses the Ore Dictionary to create it's recipes.

Maybe try an Ore Dictionary dump (NEI or minetweaker) when bdew mods are in, and TE isn't in the pack.

**Comment by accessdenied0** [comment]
I hope this Issue is still in investigation?
I recently started up my world and got the same issue I was loading the world and it was falling back into the login-screen. This part here was left in my console: http://pastebin.com/FpRz7ude
Now the strange part: I changed NOTHING. I loaded up my world like everyday.

I just couldn't explain what it causes... I hope you guys know more,

access_denied
